mlox_base_legacy.txt

```diff
@@ -34406,7 +34406,7 @@ Better Banner Signs + Expanded Sounds.esp
  Download DeJuan N. Onley's "Morrowind Compatibility Patches" from Morrowind Nexus - http://www.nexusmods.com/morrowind/mods/43880/?
  (Ref: "DNO MW Patches Readme.txt")
 MW Containers Animated + Adamantium Armor.esp
-[ALL [ANY adamantiumarmor.esp
+[ALL [ANY [SIZE 6464 adamantiumarmor.esp]
```

Comment:
I took a look at the Adamantium Armor plugin and MW Containers Animated. The only incompatibility is that MW Containers Animated swaps the model on and adds a script to the container 'de_p_chest_02_meldor' which is a chest that Adamantium Armor adds armor to. This conflict is easily solved by TES3Merge so this rule can just be removed.

Reply:
On closer inspection I realize this is the wrong base file. You can just revert any changes to it and either remove the patch rule in mlox_base.txt in another commit or leave it and I'll remove it.